python/semantic_kernel/agents/open_ai/responses_agent_thread_actions.py

```diff
@@ -719,6 +721,25 @@ def _prepare_chat_history_for_request(
                             "call_id": content.call_id,
                         }
                         response_inputs.append(rfrc_dict)
+                    case BinaryContent() if content.can_read:
```

Comment:
Could you please add a sample or two to the [samples/concepts/agents/openai_responses](https://github.com/microsoft/semantic-kernel/tree/main/python/samples/concepts/agents/openai_responses) dir showing how a dev would use/implement this? Once a sample exists, it also needs to be tracked via the [README](https://github.com/microsoft/semantic-kernel/blob/main/python/samples/concepts/README.md). Thanks!